src/Shared/Components/CICDHistory/DeploymentHistoryConfigDiff/DeploymentHistoryConfigDiffCompare.tsx

```diff
@@ -56,11 +67,37 @@ export const DeploymentHistoryConfigDiffCompare = ({
         }
     }, [])
 
+    // METHODS
+    const fetchDeploymentHistory = async (paginationOffset: number) => {
+        setTriggerHistory({ ...triggerHistory, isLoading: true })
+
+        try {
+            const { result } = await getTriggerHistory({
+                appId: +appId,
+                envId: +envId,
+                pagination: { offset: paginationOffset, size: paginationSize },
+            })
+            const nextTriggerHistory = new Map((result.cdWorkflows || []).map((item) => [item.id, item]))
+            const updatedTriggerHistory = new Map([...triggerHistory.data, ...nextTriggerHistory])
+
+            setTriggerHistory({
+                isLoading: false,
+                data: updatedTriggerHistory,
+                hasMore: result.cdWorkflows?.length === paginationSize,
+            })
+            setPipelineDeployments(getPipelineDeployments(updatedTriggerHistory))
+        } catch {
+            setTriggerHistory({ ...triggerHistory, isLoading: false })
+        }
+    }
```

Comment:
Pls check the case for release. Can discuss with Abhishek if required

Reply:
verified